files/synchronize.py

```diff
@@ -261,6 +266,7 @@ def main():
             partial = dict(default='no', type='bool'),
             verify_host = dict(default='no', type='bool'),
             mode = dict(default='push', choices=['push', 'pull']),
+            ssh_passwd = dict(default=None),
```

Comment:
Update this to include no_log option, and you might want to standardize to login_password var name.
ansible uses login_password heuristics filter no ensure that it is not logged.

Will allow this to be used with https://github.com/ansible/ansible/pull/12704 .